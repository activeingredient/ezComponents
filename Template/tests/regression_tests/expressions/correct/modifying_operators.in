{var $foo = 42, $bar = 101, $bar2 = 201, $arr = array( 'foo' => 401 )}
0001:[{$bar = 101}{$bar2 = 201}{$bar = 5}{debug_dump( $bar )},{debug_dump( $bar2 )}]
0002:[{$bar = 101}{$bar2 = 201}{$bar = $bar2 = 5}{debug_dump( $bar )},{debug_dump( $bar2 )}]
0003:[{$bar = 101}{$bar2 = 201}{$bar = $bar2 += 5}{debug_dump( $bar )},{debug_dump( $bar2 )}]
0004:[{$bar = 101}{$bar2 = 201}{$bar = $bar2 -= 5}{debug_dump( $bar )},{debug_dump( $bar2 )}]
0005:[{$bar = 101}{$bar2 = 201}{$bar = $bar2 *= 5}{debug_dump( $bar )},{debug_dump( $bar2 )}]
0006:[{$bar = 101}{$bar2 = 201}{$bar = $bar2 /= 5}{debug_dump( $bar )},{debug_dump( $bar2 )}]
0007:[{$bar = 101}{$bar2 = 201}{$bar = $bar2 %= 5}{debug_dump( $bar )},{debug_dump( $bar2 )}]
0008:[{$bar = 101}{$bar2 = 201}{$bar = $bar2 .= 5}{debug_dump( $bar )},{debug_dump( $bar2 )}]
0009:[{$bar = 101}{$bar2 = 201}{$bar += 5}{debug_dump( $bar )},{debug_dump( $bar2 )}]
0010:[{$bar = 101}{$bar2 = 201}{$bar += $bar2 = 5}{debug_dump( $bar )},{debug_dump( $bar2 )}]
0011:[{$bar = 101}{$bar2 = 201}{$bar += $bar2 += 5}{debug_dump( $bar )},{debug_dump( $bar2 )}]
0012:[{$bar = 101}{$bar2 = 201}{$bar += $bar2 -= 5}{debug_dump( $bar )},{debug_dump( $bar2 )}]
0013:[{$bar = 101}{$bar2 = 201}{$bar += $bar2 *= 5}{debug_dump( $bar )},{debug_dump( $bar2 )}]
0014:[{$bar = 101}{$bar2 = 201}{$bar += $bar2 /= 5}{debug_dump( $bar )},{debug_dump( $bar2 )}]
0015:[{$bar = 101}{$bar2 = 201}{$bar += $bar2 %= 5}{debug_dump( $bar )},{debug_dump( $bar2 )}]
0016:[{$bar = 101}{$bar2 = 201}{$bar += $bar2 .= 5}{debug_dump( $bar )},{debug_dump( $bar2 )}]
0017:[{$bar = 101}{$bar2 = 201}{$bar -= 5}{debug_dump( $bar )},{debug_dump( $bar2 )}]
0018:[{$bar = 101}{$bar2 = 201}{$bar -= $bar2 = 5}{debug_dump( $bar )},{debug_dump( $bar2 )}]
0019:[{$bar = 101}{$bar2 = 201}{$bar -= $bar2 += 5}{debug_dump( $bar )},{debug_dump( $bar2 )}]
0020:[{$bar = 101}{$bar2 = 201}{$bar -= $bar2 -= 5}{debug_dump( $bar )},{debug_dump( $bar2 )}]
0021:[{$bar = 101}{$bar2 = 201}{$bar -= $bar2 *= 5}{debug_dump( $bar )},{debug_dump( $bar2 )}]
0022:[{$bar = 101}{$bar2 = 201}{$bar -= $bar2 /= 5}{debug_dump( $bar )},{debug_dump( $bar2 )}]
0023:[{$bar = 101}{$bar2 = 201}{$bar -= $bar2 %= 5}{debug_dump( $bar )},{debug_dump( $bar2 )}]
0024:[{$bar = 101}{$bar2 = 201}{$bar -= $bar2 .= 5}{debug_dump( $bar )},{debug_dump( $bar2 )}]
0025:[{$bar = 101}{$bar2 = 201}{$bar *= 5}{debug_dump( $bar )},{debug_dump( $bar2 )}]
0026:[{$bar = 101}{$bar2 = 201}{$bar *= $bar2 = 5}{debug_dump( $bar )},{debug_dump( $bar2 )}]
0027:[{$bar = 101}{$bar2 = 201}{$bar *= $bar2 += 5}{debug_dump( $bar )},{debug_dump( $bar2 )}]
0028:[{$bar = 101}{$bar2 = 201}{$bar *= $bar2 -= 5}{debug_dump( $bar )},{debug_dump( $bar2 )}]
0029:[{$bar = 101}{$bar2 = 201}{$bar *= $bar2 *= 5}{debug_dump( $bar )},{debug_dump( $bar2 )}]
0030:[{$bar = 101}{$bar2 = 201}{$bar *= $bar2 /= 5}{debug_dump( $bar )},{debug_dump( $bar2 )}]
0031:[{$bar = 101}{$bar2 = 201}{$bar *= $bar2 %= 5}{debug_dump( $bar )},{debug_dump( $bar2 )}]
0032:[{$bar = 101}{$bar2 = 201}{$bar *= $bar2 .= 5}{debug_dump( $bar )},{debug_dump( $bar2 )}]
0033:[{$bar = 101}{$bar2 = 201}{$bar /= 5}{debug_dump( $bar )},{debug_dump( $bar2 )}]
0034:[{$bar = 101}{$bar2 = 201}{$bar /= $bar2 = 5}{debug_dump( $bar )},{debug_dump( $bar2 )}]
0035:[{$bar = 101}{$bar2 = 201}{$bar /= $bar2 += 5}{debug_dump( $bar )},{debug_dump( $bar2 )}]
0036:[{$bar = 101}{$bar2 = 201}{$bar /= $bar2 -= 5}{debug_dump( $bar )},{debug_dump( $bar2 )}]
0037:[{$bar = 101}{$bar2 = 201}{$bar /= $bar2 *= 5}{debug_dump( $bar )},{debug_dump( $bar2 )}]
0038:[{$bar = 101}{$bar2 = 201}{$bar /= $bar2 /= 5}{debug_dump( $bar )},{debug_dump( $bar2 )}]
0039:[{$bar = 101}{$bar2 = 201}{$bar /= $bar2 %= 5}{debug_dump( $bar )},{debug_dump( $bar2 )}]
0040:[{$bar = 101}{$bar2 = 201}{$bar /= $bar2 .= 5}{debug_dump( $bar )},{debug_dump( $bar2 )}]
0041:[{$bar = 101}{$bar2 = 201}{$bar %= 5}{debug_dump( $bar )},{debug_dump( $bar2 )}]
0042:[{$bar = 101}{$bar2 = 201}{$bar %= $bar2 = 5}{debug_dump( $bar )},{debug_dump( $bar2 )}]
0043:[{$bar = 101}{$bar2 = 201}{$bar %= $bar2 += 5}{debug_dump( $bar )},{debug_dump( $bar2 )}]
0044:[{$bar = 101}{$bar2 = 201}{$bar %= $bar2 -= 5}{debug_dump( $bar )},{debug_dump( $bar2 )}]
0045:[{$bar = 101}{$bar2 = 201}{$bar %= $bar2 *= 5}{debug_dump( $bar )},{debug_dump( $bar2 )}]
0046:[{$bar = 101}{$bar2 = 201}{$bar %= $bar2 /= 5}{debug_dump( $bar )},{debug_dump( $bar2 )}]
0047:[{$bar = 101}{$bar2 = 201}{$bar %= $bar2 %= 5}{debug_dump( $bar )},{debug_dump( $bar2 )}]
0048:[{$bar = 101}{$bar2 = 201}{$bar %= $bar2 .= 5}{debug_dump( $bar )},{debug_dump( $bar2 )}]
0049:[{$bar = 101}{$bar2 = 201}{$bar .= 5}{debug_dump( $bar )},{debug_dump( $bar2 )}]
0050:[{$bar = 101}{$bar2 = 201}{$bar .= $bar2 = 5}{debug_dump( $bar )},{debug_dump( $bar2 )}]
0051:[{$bar = 101}{$bar2 = 201}{$bar .= $bar2 += 5}{debug_dump( $bar )},{debug_dump( $bar2 )}]
0052:[{$bar = 101}{$bar2 = 201}{$bar .= $bar2 -= 5}{debug_dump( $bar )},{debug_dump( $bar2 )}]
0053:[{$bar = 101}{$bar2 = 201}{$bar .= $bar2 *= 5}{debug_dump( $bar )},{debug_dump( $bar2 )}]
0054:[{$bar = 101}{$bar2 = 201}{$bar .= $bar2 /= 5}{debug_dump( $bar )},{debug_dump( $bar2 )}]
0055:[{$bar = 101}{$bar2 = 201}{$bar .= $bar2 %= 5}{debug_dump( $bar )},{debug_dump( $bar2 )}]
0056:[{$bar = 101}{$bar2 = 201}{$bar .= $bar2 .= 5}{debug_dump( $bar )},{debug_dump( $bar2 )}]
