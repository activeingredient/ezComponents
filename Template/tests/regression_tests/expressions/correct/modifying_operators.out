0001:[5,201]
0002:[5,5]
0003:[206,206]
0004:[196,196]
0005:[1005,1005]
0006:[40.2,40.2]
0007:[1,1]
0008:['2015','2015']
0009:[106,201]
0010:[106,5]
0011:[307,206]
0012:[297,196]
0013:[1106,1005]
0014:[141.2,40.2]
0015:[102,1]
0016:[2116,'2015']
0017:[96,201]
0018:[96,5]
0019:[-105,206]
0020:[-95,196]
0021:[-904,1005]
0022:[60.8,40.2]
0023:[100,1]
0024:[-1914,'2015']
0025:[505,201]
0026:[505,5]
0027:[20806,206]
0028:[19796,196]
0029:[101505,1005]
0030:[4060.2,40.2]
0031:[101,1]
0032:[203515,'2015']
0033:[20.2,201]
0034:[20.2,5]
0035:[0.49029126213592,206]
0036:[0.51530612244898,196]
0037:[0.10049751243781,1005]
0038:[2.5124378109453,40.2]
0039:[101,1]
0040:[0.050124069478908,'2015']
0041:[1,201]
0042:[1,5]
0043:[101,206]
0044:[101,196]
0045:[101,1005]
0046:[21,40.2]
0047:[0,1]
0048:[101,'2015']
0049:['1015',201]
0050:['1015',5]
0051:['101206',206]
0052:['101196',196]
0053:['1011005',1005]
0054:['10140.2',40.2]
0055:['1011',1]
0056:['1012015','2015']
